backend/src/baserow/api/workspaces/users/serializers.py

```diff
@@ -41,6 +44,13 @@ def get_name(self, object):
     def get_email(self, object):
         return object.user.email
 
+    def get_two_factor_auth(self, object):
+        return TwoFactorAuthSerializer(
+            object.user.two_factor_auth_providers.first().specific
```

Comment:
It looks like the .first() is causing N number of queries. Would it be possible to just use .all()[0] and maybe prefetch the specific object earlier? That should reduce the number of queries.